coderd/coderdtest/coderdtest.go

```diff
@@ -56,14 +56,15 @@ import (
 )
 
 type Options struct {
-	AWSCertificates      awsidentity.Certificates
-	Authorizer           rbac.Authorizer
-	AzureCertificates    x509.VerifyOptions
-	GithubOAuth2Config   *coderd.GithubOAuth2Config
-	GoogleTokenValidator *idtoken.Validator
-	SSHKeygenAlgorithm   gitsshkey.Algorithm
-	APIRateLimit         int
-	AutobuildTicker      <-chan time.Time
+	AWSCertificates       awsidentity.Certificates
+	Authorizer            rbac.Authorizer
+	AzureCertificates     x509.VerifyOptions
+	GithubOAuth2Config    *coderd.GithubOAuth2Config
+	GoogleTokenValidator  *idtoken.Validator
+	SSHKeygenAlgorithm    gitsshkey.Algorithm
+	APIRateLimit          int
+	AutobuildTicker       <-chan time.Time
+	AutobuildStatsChannel chan<- executor.RunStats
```

Comment:
I think we could call this `AutobuildStats`. The fact that it's a channel doesn't seem very important to the caller in this context.